src/components/pages/Registration/RegisterStep3.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import RegistrationProgress from './RegistrationProgress';
+import StudentForm from './RegisterFormComponents/StudentForm';
+import ParentAccountInfo from './RegisterFormComponents/ParentAccountInfo';
+import '../../../styles/registration.less';
+
+const numOfStudents = Number(localStorage.getItem('number of students'));
+
+const initialStudentArr = new Array(numOfStudents).fill({
+  name: '',
+  pronoun: '',
+  age: '',
+  grade: '',
+  studentEmail: '',
+  avatarURL: '',
+  parent_id: '',
+});
+
+const initialParentInfo = {
+  name: '',
+  email: '',
+  location: '',
+  phone: '',
+  notes: '',
+  parent_id: '',
+  role_id: 3,
+  children: [],
+};
+
+export default function() {
+  const [studentArr, setStudentArr] = useState(initialStudentArr);
+  const [parentInfo, setParentInfo] = useState(initialParentInfo);
+
+  // send accountDetails to the backend
+  const accountDetails = {
```

Comment:
This variable can probably be stashed for now, since you do not have the handleSubmit ready yet.